_positions/product-lead.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Product Lead
+permalink: pay-grades/product-lead/
+team: 18F
+active: true
+---
+**GS Grade Level:** 13
```

Comment:
thank you! there are some lingering questions about the approved job posting so i'm waiting for some answers